### ISSUE TITLE
consider documenting a customization example

### ISSUE THREAD

**Issue by acerbic-cover** [bbatsov/solarized-emacs#190]
Motivated by the same sort of sentiment expressed in #158, I have written an extension theme where I would like to use the solarized approach, but tweak certain faces.

Alas, I am not an elisp hacker, so my ability to intuit what elisp should go in the [optional child theme](https://github.com/bbatsov/solarized-emacs/blob/master/solarized.el#L151) function is limited.

I followed the pattern to create a custom-theme; that part is working fine.  I can invoke

    M-x load-theme

choose my customized `solarized-jwm-dark` theme and my extension function gets called.

However, I am having a hard time guessing what the idioms are to further customize the theme.
I hacked around guessing at how to override a sample face for, e.g., `font-lock-string-face`, without much success.  For example, here is one of the approaches I tried:

```elisp
(defun jwm-customize-solarize-dark ()
  "apply some of jwms color customizations on top of the solarized dark theme"

  (let* ((class '((class color) (min-colors 89)))
         (light-coral    "LightCoral"))
    (custom-theme-set-faces
     'solarized-jwm-dark
     `(font-lock-string-face ((,class (:foreground ,light-coral))))
     )))
```

As I said, I was just pattern matching and guessing as to how to structure an extension function.

Can someone point me in the right direction here?  I would be happy to supply the result back to the project in a pull request to document for the future as appropriate.

**Comment by thomasf** [comment]
Oh.. I think it might have been there before.. I had the impression that the child theme system had stopped working so I might have removed the example.. Will take a quick look.

**Comment by thomasf** [comment]
there you go https://github.com/bbatsov/solarized-emacs/tree/master/child-theme-example

**Comment by acerbic-cover** [comment]
Groovy.  That will help me no doubt.  Thx

**Comment by thomasf** [comment]
I just fixed some copy/paste errors...

**Comment by acerbic-cover** [comment]
@thomasf I only see one commit [3819bda](https://github.com/bbatsov/solarized-emacs/commit/3819bdaf1addc2d4a9857eef9a087c3974e91c94)

I see you pushed another change for #158 as well.

In any case, I put your code into my function:

```elisp
(defun jwm-customize-solarize-dark ()
  "apply some of jwms color customizations on top of the solarized dark theme"

  (message "yup, running jwm-customize-solarize-dark")
  (custom-theme-set-faces
   theme-name
   `(font-lock-string-face ((,class (:foreground ,red :background ,base03))))
   ))
```

and while I see the message `yup ...`, `describe-face` says `font-lock-string-face` is not changed:

```
Defined in `font-lock.el'.
  ...
       Foreground: #2aa198
DistantForeground: unspecified
       Background: unspecified
```

I would expect to see different values for foreground and background there.

Any pointers here appreciated.

**Comment by thomasf** [comment]
I rebased and force pushed the change..

I have the feeling that child theme still might not work as expected for all situations.. I cant recall what was wrong with that feature..

I also added the defcustom (fixed #158) to set the doc face to violet if you can do with that for the time being.
